ARTstor.js

```diff
@@ -1,131 +1,484 @@
+            portal = 'default';
+        }
+        processPortalData(doc, dataItem, json, portalMap[portal], portal);
+        getNotesDataItem(url, objItem, dataItem);
+    });
+}
+
+/**
+	This procedure process the json, and add the json value to the 
+	Zotero item.
+**/
+function processPortalData(doc, dataItem, json, fieldMap, portal) {
+    var fieldName;
+    var fieldValue;
+    if (portal == 'archaeology') {
+        var hasSiteName = false;
+        for (var i = 0; i < json.metaData.length; i++) {
+            fieldName = json.metaData[i].fieldName;
+            fieldValue = htmlDecode(doc, json.metaData[i].fieldValue);
+            if (fieldName in fieldMap) {
+                var key = fieldMap[fieldName];
+                if (fieldName == 'Site Name') {
+                    hasSiteName = true;
+                    setItemValue(dataItem, "title", fieldValue);
+                } else if (fieldName == 'Artifact Title') {
+                    if (hasSiteName) {
+                        setItemLabelValue(doc, dataItem, "extra", fieldName, dataItem.title);
+                        hasSiteName = false;
+                    }
+                    setItemValue(dataItem, "title", fieldValue);
+                } else {
+                    setItemValue(dataItem, key, fieldValue);
+                }
+            } else {
+                setItemLabelValue(doc, dataItem, "extra", fieldName, fieldValue);
+            }
+        }
+
+    } else {
+        for (var i = 0; i < json.metaData.length; i++) {
+            fieldName = json.metaData[i].fieldName;
+            fieldValue = htmlDecode(doc, json.metaData[i].fieldValue);
+            // fieldValue = json.metaData[i].fieldValue;
+            if (fieldName in fieldMap) {
+                var key = fieldMap[fieldName];
+                if (key == 'creators') {
+                    setItemCreator(dataItem, fieldValue);
+                } else {
+                    setItemValue(dataItem, key, fieldValue);
+                }
+            } else {
+                setItemLabelValue(doc, dataItem, "extra", fieldName, fieldValue);
+            }
+        }
+    }
+    if (json.SSID !== undefined && json.SSID !== "") {
+        setItemLabelValue(doc, dataItem, "extra", "SSID", json.SSID);
+    }
+    if (dataItem.title == undefined) {
+        dataItem.title = "Unknown";
+    }
+}
+
+function setItemCreator(dataItem, fieldValue) {
+    var names = [];
+    if (fieldValue.indexOf(';') > 0) {
+        names = fieldValue.split(';')
+    } else {
+        names.push(fieldValue);
+    }
+    for (var i = 0; i < names.length; i++) {
+        var str = names[i];
+        var contributor = "author";
+        var name = str;
+        var value = name.replace(/<\/?[^>]+(>|$)/g, " ").replace(/(&gt;)|(&lt;)/g, "");
+        dataItem.creators.push(ZU.cleanAuthor(value, contributor, false));
+    }
+}
+
+function cleanStringValue(str) {
+    var cleanValue = str.replace(/\<wbr\/>/g, "");
```

Comment:
That looks fishy, because you escaped `<` but not `>`. I think you don't have to escape any of these.